demisto_sdk/commands/content_graph/parsers/integration_script.py

```diff
@@ -26,6 +31,23 @@ def __init__(
         super().__init__(path, pack_marketplaces, git_sha=git_sha)
         self.script_info: Dict[str, Any] = self.yml_data.get("script", {})
         self.connect_to_api_modules()
+        self.structure_errors = self.validate_structure()
+
+    @property
+    def strict_obj(self):
+        return self.__class__
```

Comment:
is this right?

Reply:
Yes, we had to change it to a regular method instead of an abstract one

Reply:
let's discuss it offline